python/lsst/meas/algorithms/sourceSelector.py

```diff
@@ -97,3 +103,301 @@ def _isBad(self, source):
 sourceSelectorRegistry = pexConfig.makeRegistry(
     doc="A registry of source selectors (subclasses of BaseSourceSelectorTask)",
 )
+
+
+class ColorLimit(pexConfig.Config):
+    """Select sources using a color limit
+
+    This object can be used as a `lsst.pex.config.Config` for configuring
+    the limit, and then the `apply` method can be used to test sources in
```

Comment:
"...can be used to identify sources in the catalog that match the configured limit.", or something like that. (and for each one below)